src/includes/getting-started-primer/dart.mdx

```diff
@@ -1,5 +1,16 @@
 <Note>
 
-Pure Dart SDK used by any Dart application like AngularDart, CLI and Server, it enables reporting messages and errors.
+Sentry's Dart SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/). For Flutter apps, refer to [our Flutter documentation](/platforms/flutter) instead.
 
 </Note>
+
+**Features:**
+
+- Events [enriched](/platforms/dart/enriching-events/context/) with device data
+- Breadcrumbs automatically captured:
+  - by the [Dart SDK](/platforms/dart/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+  - as well as `http` with the [Dart SDK](/platforms/dart/)
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
```

Comment:
```suggestion
```

This is Flutter only, maybe we need to add `dart` as non supported for Release health and let only Flutter